UnitTests/BraintreeVenmoTests/BTVenmoRequest_Tests.swift

```diff
@@ -18,4 +18,25 @@ class BTVenmoRequest_Tests: XCTestCase {
         XCTAssertEqual(request.collectCustomerShippingAddress, false)
         XCTAssertEqual(request.collectCustomerBillingAddress, false)
     }
+
+    func testIsFinalAmount_whenIsFinalAmountSetAsTrue_returnsTrue() {
+        let request = BTVenmoRequest(paymentMethodUsage: .singleUse)
+        request.isFinalAmount = true
+        
+        XCTAssertEqual(request.isFinalAmount, true)
+
+    }
+
+    func testIsFinalAmount_whenIsFinalAmountSetAsFalse_returnsFalse() {
+        let request = BTVenmoRequest(paymentMethodUsage: .singleUse)
+        request.isFinalAmount = false
+
+        XCTAssertEqual(request.isFinalAmount, false)
+
+    }
```

Comment:
I think the test you added on line 38 is sufficient. The 2 above don't validate any logic 

```suggestion
```

Reply:
✅ : https://github.com/braintree/braintree_ios/pull/1192/commits/a191a8f021ebeb397de8a556c0b0d61b308762bb